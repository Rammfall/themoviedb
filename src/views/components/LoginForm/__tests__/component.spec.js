import React from 'react'
import { shallow } from 'enzyme'

import LoginFormComponent from '../component'

describe('LoginForm match snapshot', () => {
  it('with default props', () => {
    jest.mock('react-intl')

    const component = shallow(<LoginFormComponent />)

    expect(component).toMatchSnapshot()
  })
})
