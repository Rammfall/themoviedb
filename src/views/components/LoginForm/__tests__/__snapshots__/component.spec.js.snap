// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginForm match snapshot with default props 1`] = `
<div
  className="center"
>
  <Layout>
    <Content>
      <Row
        justify="center"
        type="flex"
      >
        <Col>
          <Form>
            <Title>
              <FormattedMessage
                id="application.theMovieDB"
              />
            </Title>
            <Field
              component={[Function]}
              name="username"
              placeholder="Username"
              prefix={
                <ForwardRef(UserOutlined)
                  style={
                    Object {
                      "color": "rgba(0,0,0,.25)",
                    }
                  }
                />
              }
            />
            <Field
              component={[Function]}
              name="password"
              placeholder="Password"
              prefix={
                <ForwardRef(LockOutlined)
                  style={
                    Object {
                      "color": "rgba(0,0,0,.25)",
                    }
                  }
                />
              }
              type="password"
            />
            <FormItem>
              <Button
                htmlType="submit"
                type="primary"
              >
                Log in
              </Button>
            </FormItem>
          </Form>
        </Col>
      </Row>
    </Content>
  </Layout>
</div>
`;
